.cascade-code/Chapman.edu/_components/omni_nav_v2/omni_nav.vtl

```diff
@@ -112,7 +112,14 @@
     ## Is current page directory $pagePathDir under this $navDirectory? If so,
     ## we found our $navDirectory.
     #if ( $pagePath.contains($dirName) && $pagePathDir == $dirName )
```

Comment:
The other ways I could come up with to do this were much more computationally expensive than this one. I'm open to discussion.

Reply:
I'm good with this.